site/src/layouts/DesignerBio.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import Link from 'gatsby-link'
+import styled from 'emotion/react'
+import cx from 'classnames'
+import { sansfont, childLink, Header2 } from './emotion-base'
+
+const Container = styled.section`margin-top: 60px;`
+
+const Bio = styled.div`
+  composes: ${childLink}, ${sansfont};
+  font-size: 16px;
+  line-height: 1.4;
+`
+
+const DesignerProjects = ({ bio }) => {
+  // TODO: bio should just be full markdown in the CMS?
```

Comment:
Yeah I think that would be best - seems like a common thing to edit and u would want a nice interface

Reply:
cool !